src/DcGeneral/Data/FilterBuilder.php

```diff
@@ -295,17 +296,19 @@ protected function buildNativeSqlProcedure(FilterBuilderSql $procedure, $childre
             // Try to parse the sub procedure and extract as much as possible.
             if (($child['operation'] == 'AND') || ($child['operation'] == 'OR')) {
                 $subProcedure = new FilterBuilderSql($tableName, $child['operation'], $this->connection);
-                $subSkipped   = $this->buildNativeSqlProcedure($subProcedure, $child['children']);
```

Comment:
Couldn't we just do
```php
if (null === $child['children']) {
    continue;
}
```
here?
This would omit indenting everything here one level down.

Reply:
Sure! My fault. Ready for review.